test/tests/pbs_smoketest.py

```diff
@@ -750,13 +751,19 @@ def test_round_robin(self):
         a = {'scheduling': 'False'}
         self.server.manager(MGR_CMD_SET, SERVER, a)
         self.server.expect(JOB, {'job_state=R': 9})
-        cycle = self.scheduler.cycles(start=start_time, end=int(time.time()))
+        end_time = int(time.time()) + 1
```

Comment:
Due to the start_time and end_time being truncated with int(), the cycles() call may not read all the required logs to find all the data it needs and fail.  This might be related to this: https://pbspro.atlassian.net/browse/PP-359